mesonbuild/backend/ninjabackend.py

```diff
@@ -592,9 +592,19 @@ def generate_coverage_rules(self, outfile):
 
     def generate_install(self, outfile):
         install_data_file = os.path.join(self.environment.get_scratch_dir(), 'install.dat')
+        if self.environment.is_cross_build():
+            bins = self.environment.cross_info.config['binaries']
+            if 'strip' not in bins:
+                mlog.warning('Cross file does not specify strip binary, result will not be stripped.')
+                strip_bin = None
+            else:
+                strip_bin = mesonlib.stringintlistify(bins['strip'])
```

Comment:
`stringintlistify` will accept both `int` and `str`, we don't want to use that here. We want to use `stringlistify` here.